core/src/main/kotlin/org/evomaster/core/database/DbAction.kt

```diff
@@ -52,6 +52,10 @@ class DbAction(
         }
     }
 
+    /**
+     * whether the insertion was executed successfully
+     */
+    var insertExecutedSuccessfully = false
```

Comment:
didn't we discuss that this should not be in the action itself, but on the result? 
or was this supposed to be refactored at a later stage (but, in this, it misses a TODO comment)

Reply:
I will refactor this later. now the TODO is added.